src/bundle/ui-dev/src/modules/common/tag/tag.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon/icon';
+import { createCssClassNames } from '../helpers/css.class.names';
+
+const Tag = ({ content, onRemove, extraClasses }) => {
+    const className = createCssClassNames({
```

Comment:
If everything has true, can't it be achieved by simple template string instead of func?

Reply:
I find this notation cleaner and easier to understand.